### ISSUE TITLE
Make the status panel show more useful info

### ISSUE THREAD

**Issue by canadaduane** [jesseduffield/lazygit#2400]
**Topic**

Currently, the status panel shows a large logo of `lazygit` with copyright and help links. I think it would be more useful to show status information of the current git state.

![image](https://user-images.githubusercontent.com/129/215124611-c18cee98-20d8-435f-8189-af5fa5d06f2a.png)

**Your thoughts**

Things I'd like to know:

1. If the current branch name does not fit inside the "small status" panel (top left), I'd like to see its full name in the "large status" panel (right-hand side).

![image](https://user-images.githubusercontent.com/129/215124928-9e7b348b-604c-443a-9fac-76135a0b1e64.png)

2. Is my installed version of lazygit out of date? Has there been a new release?
3. What is the full path of this repository (locally on my filesystem)?
4. What is the URI of the "upstream" repository?

**Comment by JukMR** [comment]
Yes! I would also like these features! I think it would be good if they just get added in this panel. If needed, I could provide a pull requests doing this.
